src/test/java/space/obminyashka/items_exchange/rest/UserControllerIntegrationTest.java

```diff
@@ -58,13 +62,17 @@ class UserControllerIntegrationTest extends BasicControllerTest {
     @MockBean
     private UserServiceImpl userService;
 
+    @MockBean
+    private MailService mailService;
+
     @SpyBean
     private ImageServiceImpl imageService;
 
     @Captor
     private ArgumentCaptor<MultipartFile> captor;
 
     private static String maxPhonesAmount;
+    private static final String DOMAIN_URL = "https://obminyashka.space";
```

Comment:
That is the test with context why don't you simply use the default HOST?

Reply:
++